.github/workflows/gh-ci.yaml

```diff
@@ -30,11 +30,14 @@ env:
   HTML_DIR: "doc/build/html"
 
 jobs:
-  build_docs:
-    runs-on: ubuntu-latest
-    env:
-      CYTHON_TRACE_NOGIL: 1
-      MPLBACKEND: agg
+  tests:
+    runs-on: ${{ matrix.OS }}-latest
+    name: "examples-tests"
+    strategy:
+      fail-fast: False
+      matrix:
+        os: ['ubuntu']
+        python-version: [3.8]
```

Comment:
Why a matrix if there's only one entry?

Reply:
my bad, I did all of this before your latest commits and it seems `git rebase` just overwrote the file without showing any conflicts.